spec/system/task_spec.rb

```diff
@@ -98,15 +93,16 @@
   end
 
   describe 'Task削除' do
+
+    let!(:task){create(:task, :task_date)}
```

Comment:
書き方の話なので好みが分かれるところですが、letの直前は改行せずに詰めてしまっても良い気がしますね。
```suggestion
    let!(:task){ create(:task, :task_date) }
```

Reply:
d1841bf でletの前の行をつめる修正を行いました。

[![Image from Gyazo](https://i.gyazo.com/12fb58056da3676024f856d69d2c7fb3.png)](https://gyazo.com/12fb58056da3676024f856d69d2c7fb3)
こちらのlet!の定義については前の行をつめると確認しにくくなってしまうのではと思い1行空けました。